doc/media/openage/modpack_definition_file.md

```diff
@@ -0,0 +1,109 @@
+# Modpack Definition File
+
+The modpack definition file is the header file for [modpacks](modpacks.md).
+
+TOML is used as the configuration language.
+
+This document contains information about the parameters and sections that
+can be interpreted by the engine.
+
+## File Header
+
+The file starts with the comment:
+
+```
+# openage modpack definition file
+```
+
+Additionally, the following parameters have to be specified
+
+Parameter            | Data Type | Optional | Description
+---------------------|-----------|----------|------------
+`file_version`       | String    | No       | Version of the modpack definition file format.
+
+
+## [info] Section
+
+`[info]` contains general information about the modpack.
+
+Parameter          | Data Type     | Optional | Description
+-------------------|---------------|----------|------------
+`packagename`      | String        | No       | Name of the modpack.
+`version`          | String        | No       | Version number.
+`repo`             | String        | Yes      | Name of the repo where the package comes from.
+`alias  `          | String        | Yes      | Alias of the modpack.
```

Comment:
more spaces

Reply:
the description should mention that aliases can be used to replace other modpacks.

Reply:
fixed